src/explore-education-statistics-frontend/src/components/Page.tsx

```diff
@@ -21,13 +42,22 @@ const Page = ({ children, breadcrumbs = [], wide = false }: Props) => {
         })}
       >
         <PageBanner />
-        <Breadcrumbs breadcrumbs={breadcrumbs} />
+        <Breadcrumbs
+          breadcrumbs={
+            isHomepage
+              ? undefined
+              : breadcrumbs.concat([{ name: breadcrumbLabel || title }])
+          }
+        />
 
         <main
           className="govuk-main-wrapper app-main-class"
           id="main-content"
           role="main"
         >
+          {!hideTitle && (
```

Comment:
Similar thing to the redundant `hideCaption` prop here. If we don't want to show a title, then we can just pass an empty stringr.